titus-server-master/src/main/java/com/netflix/titus/master/agent/service/DefaultAgentManagementService.java

```diff
@@ -182,6 +172,16 @@ public Completable updateInstanceGroupAttributes(String instanceGroupId, Map<Str
                 }).toCompletable();
     }
 
+    @Override
+    public Completable updateAgentInstanceAttributes(String instanceId, Map<String, String> attributes) {
```

Comment:
This is a full override. I think we should also provide merge/remove operations.

Reply:
Yes, I believe we should remove this API and move to the style of add/remove attributes to be safer for multiple concurrent updates.